test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,8 +60,7 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
-    private void assertAreUnique(Object... objects) {
-        assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
+    private void assertAreUnique(Object... objects) { assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation here. Put the code below method name & curly bracket.